spec/spec_helper.rb

```diff
@@ -16,6 +40,10 @@
   config.ignore_hosts('codeclimate.com') if ENV['CI']
   config.cassette_library_dir = File.join(ManageIQ::Providers::IbmPowerHmc::Engine.root, 'spec/vcr_cassettes')
 
+  config.before_record do |i|
```

Comment:
Super-minor, I think there's a way to do this with `config.filter_sensitive_data` that might be a little more elegant. See https://relishapp.com/vcr/vcr/v/6-1-0/docs/configuration/filter-sensitive-data#filter-dynamic-data-based-on-yielded-http-interaction